packages/zosjobs/__tests__/__system__/ModifyJobs.system.test.ts

```diff
@@ -34,47 +33,46 @@ describe("Modify Jobs - System Tests", () => {
         systemProps = testEnvironment.systemTestProperties;
         REAL_SESSION = TestEnvironment.createZosmfSession(testEnvironment);
         account = systemProps.tso.account;
-        jobname = testEnvironment.systemTestProperties.zosmf.user;
         jobclass = testEnvironment.systemTestProperties.zosjobs.jobclass;
         modifiedJobClass = testEnvironment.systemTestProperties.zosjobs.modifiedJobclass;
```

Comment:
Should we do something about the new properties added to the `default_parameters.yaml`?

For example, if it's missing from the `custom_...`, should we try to pull them from the `default_...`?
One way we could do it is in the `testEnvironment`... we could merge the systemTestProperties from `{...default, ...custom}` 😋